frontend/src/metabase/query_builder/components/view/QuestionNotebookButton/QuestionNotebookButton.styled.tsx

```diff
@@ -0,0 +1,15 @@
+import styled from "@emotion/styled";
+import Button from "metabase/core/components/Button";
+import { color } from "metabase/lib/colors";
+
+interface ButtonRootProps {
+  isSelected?: boolean;
+}
+
+export const ButtonRoot = styled(Button)<ButtonRootProps>`
+  color: ${props => !props.isSelected && color("text-dark")};
+
+  &:hover {
+    color: ${props => props.isSelected && color("brand")};
```

Comment:
I think this should be inversed, see https://github.com/metabase/metabase/pull/32514/commits/58e5fc6e80246f612bec90dd8d63fc7da1afdef6

```suggestion
    color: ${props => !props.isSelected && color("brand")};
```